### PR TITLE
Adds description about various options available with the acts_as_list method

### PR DESCRIPTION
**Pull request by udit7590**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,15 @@ class TodoItem < ActiveRecord::Base
 end
 ```
 
+## More Options
+You can pass the following options with **acts_as_list** class method
```
Comment (swanandp) [inline README.md]: No need for this.

File: README.md [hunk 2]
```diff
@@ -85,6 +85,15 @@ class TodoItem < ActiveRecord::Base
...
+- `column`
+By default 'position'. Use this option if the column name in your database is different from position.
```
Comment (swanandp) [inline README.md]: Just say `default: 'position'`

**Comment by swanandp** [comment]
This documentation is best left to the code. By putting it in README, we're only repeating ourselves.

**Comment by udit7590** [comment]
I think README file is to help developers understand how to use the gem.
I faced an issue where I was not able to determine what is the default value that is being picked up by position cloumn when I was saving my records. When I finally looked at the code, I realized that I can actually use the option `add_new_at` available with the acts_as_list method. If it would have been present in README file it would have saved a lot of time for me.
Hence I believe README file should tell about all the behavior and options available with the gem to help developers understand how wisely they could use the gem.

**Comment by brendon** [comment]
I kind of agree with @udit7590, it is handy to have the options documented in the README. Up to you though Swanand. :)

**Comment by akshay-vishnoi** [comment]
:+1:

**Comment by swanandp** [comment]
Alright, let's put it in.  @udit7590 Can you make the fix I commented on, and then squash the two commits? I am generally not in favour of squashing, but the commits here are tiny enough to warrant a squash.

**Comment by udit7590** [comment]
@swanandp Squashed and fixed the changes.

**Comment by swanandp** [comment]
Alright, merging this in.

**Pull request merged**